pants

```diff
@@ -124,8 +122,34 @@ at https://raw.githubusercontent.com/Eric-Arellano/setup/0d445edef57cb89fd830db7
   echo "${pants_version}"
 }
 
+function set_supported_python_versions {
```

Comment:
Returning `Tuple[List[str], List[str], str]` was far too complex in Bash, so mutation is the best I could come up with.